docs/modules/preprocessing.rst

```diff
@@ -39,4 +39,14 @@ the data with clarity. To better understand the data, we need to use
 *dimensionality reduction* methods that can reduce the number of features
 while still preserving the most relevant information.
 :doc:`Here <preprocessing/dim_reduction>` you can learn more about the
-dimension reduction methods available in the library.
\ No newline at end of file
+dimension reduction methods available in the library.
+
+Feature construction
+--------------------
+
+It is crucial to create features from the original ones so they can be
```

Comment:
"crucial" is maybe too exaggerated. It is an important technique, for sure, but not "crucial".